### ISSUE TITLE
wrangler: command not found

### ISSUE THREAD

**Issue by famzah** [cloudflare/wrangler#1632]
I've installed Wrangler following the instructions in the Dashboard:
`npm install -g @cloudflare/wrangler`

The installation seems to have completed successfully:
```
$ npm install -g @cloudflare/wrangler
/home/famzah/.node-v9.4.0-linux-x64/bin/wrangler -> /home/famzah/.node-v9.4.0-linux-x64/lib/node_modules/@cloudflare/wrangler/run-wrangler.js

> @cloudflare/wrangler@1.12.1 postinstall /home/famzah/.node-v9.4.0-linux-x64/lib/node_modules/@cloudflare/wrangler
> node ./install-wrangler.js

Downloading release https://workers.cloudflare.com/get-npm-wrangler-binary/1.12.1/x86_64-unknown-linux-musl
wrangler has been installed!
+ @cloudflare/wrangler@1.12.1
added 34 packages in 5.088s
```

However, the binary is not added in my $PATH and I get the following when executing `wrangler`:
`wrangler: command not found`

I see the following executable files with the name "wrangler" in my home directory:
```bash
/home/famzah/.wrangler/bin/wrangler # ELF 64-bit LSB executable, statically linked
/home/famzah/.node-v9.4.0-linux-x64/bin/wrangler -> /home/famzah/.node-v9.4.0-linux-x64/lib/node_modules/@cloudflare/wrangler/run-wrangler.js # a tiny JavaScript
```

Both binaries seem to do the same -- they invoke the Wrangler CLI and show the following version info:
`👷 ✨  wrangler 1.12.0`

I've got the following questions:
- Which of the two executables should I use for the Wrangler CLI?
- None of them was added in any of the (local) $PATH directories. What's the expected behavior, so that the command `wrangler` works?

**Comment by famzah** [comment]
This seems related to #266.

**Comment by nataliescottdavidson** [comment]
@famzah the wrangler executable should be at `/usr/local/bin/wrangler`. Have you installed `npm` with a version manager such as `nvm`? Your `npm` install can effect where packages get installed

**Comment by famzah** [comment]
Thank you @nataliescottdavidson.

It's probably my custom installation of `npm` that's causing the issue. I've downloaded the "Linux Binaries (x64)" from the official [Node.js](https://nodejs.org/en/download/) website. Then symlink'ed the binary in my $PATH:
```bash
$ ls -la ~/bin/npm
/home/famzah/bin/npm -> /home/famzah/.node-v9.4.0-linux-x64/bin/npm
```

It's logical that I also needed to add `~/.node-v9.4.0-linux-x64/bin` to my $PATH.

---

Alright, then only the other question is still open. I see the following executable files with the name "wrangler" in my home directory:
```bash
/home/famzah/.wrangler/bin/wrangler # ELF 64-bit LSB executable, statically linked
/home/famzah/.node-v9.4.0-linux-x64/bin/wrangler -> /home/famzah/.node-v9.4.0-linux-x64/lib/node_modules/@cloudflare/wrangler/run-wrangler.js # a tiny JavaScript
```

Both binaries seem to do the same -- they invoke the Wrangler CLI and show the following version info:
`👷 ✨ wrangler 1.12.0`

❓ Which of the two executables should I use for the Wrangler CLI? Why there are two executables doing the same thing?
